res/skins/Shade/effectunit.xml

```diff
@@ -15,40 +15,54 @@
 				</Style>
 				<BackPath>style/style_bg_effectunit</BackPath>
 				<Children>
-					<WidgetGroup>
-						<Pos>9,5</Pos>
-						<Size>170,90</Size>
-						<Layout>vertical</Layout>
+					<HighlightingGroup>
```

Comment:
This case here is somehow random. A HighlightingGroup Group is just a Widget that can be highlighted. It can be placed at any place where you can place a WidgetGroup as well. It is just a WidgetGroup with highlighting features. 
There is also no need to place other HighlightingGroup as siblings into a WidgetGroup. You can spread them all over the skin.  

The word "selected" has already a Qt css meaning. I did not fell in love with the word "highlight", but I picked it, because it is not used in any default Qt style pseudo states, so it should be less confusing. We can pick any other name if you a better unused one.